test_4062.py

```diff
@@ -0,0 +1,29 @@
+import setupext
```

Comment:
+ Do you think it is better not to include the test_4062.py file?
+ Is "nose" a module for testing? Can you recommend a reference? (http://pythontesting.net/framework/nose/nose-introduction/ <- This I could find)